experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -136,7 +137,19 @@ def _verify_url(request, base_url):
 
 @pytest.fixture
 def kinto_client(default_data):
-    return KintoClient(APPLICATION_KINTO_COLLECTION[default_data.application])
+    kinto_url = "http://kinto:8888/v1"
+    if not os.environ.get("DEBIAN_FRONTEND"):
+        kinto_url = "http://localhost:8888/v1"
```

Comment:
e.g. here we would do `kinto_url = os.getenv("INTEGRATION_TEST_KINTO_URL", "http://kinto:8888/v1")`